File: coupon-service-api/src/main/java/com/kakao/mobility/kmcoupon/CouponServiceApiApplication.java
package com.kakao.mobility.kmcoupon;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

import javax.annotation.PostConstruct;
import java.util.TimeZone;

@SpringBootApplication
public class CouponServiceApiApplication {

    public static void main(String[] args) {
        SpringApplication.run(CouponServiceApiApplication.class, args);
    }

    @PostConstruct
    public void started() {
        TimeZone.setDefault(TimeZone.getTimeZone("Asia/Seoul"));
    }

}

File: coupon-service-api/src/main/java/com/kakao/mobility/kmcoupon/CouponServiceApiApplication.kt
package com.kakao.mobility.kmcoupon

import org.springframework.boot.autoconfigure.SpringBootApplication
import java.util.TimeZone
import kotlin.jvm.JvmStatic
import org.springframework.boot.SpringApplication
import com.kakao.mobility.kmcoupon.CouponServiceApiApplication
import javax.annotation.PostConstruct

@SpringBootApplication
class CouponServiceApiApplication {
    @PostConstruct
    fun started() {
        TimeZone.setDefault(TimeZone.getTimeZone("Asia/Seoul"))
    }

    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            SpringApplication.run(CouponServiceApiApplication::class.java, *args)
        }
    }
}